retrofit-clients/build.gradle

```diff
@@ -8,7 +8,8 @@ dependencies {
     // ssl-config is not used in this project, but we depend on it here in order to provide
     // it transitively to users of retrofit-clients as a convenience.
     compile project(':ssl-config')
-    compile libs.retrofit
+    compile libs.retrofit.retrofit
+    compile libs.retrofit.gson
```

Comment:
yep, will do

On Thu, Jun 23, 2016 at 12:44 PM Mark Elliot <notifications@github.com>
wrote:

> In retrofit-clients/build.gradle
> <https://github.com/palantir/http-remoting/pull/143#discussion_r68301733>:
>
> > @@ -8,7 +8,8 @@ dependencies {
> >      // ssl-config is not used in this project, but we depend on it here in order to provide
> >      // it transitively to users of retrofit-clients as a convenience.
> >      compile project(':ssl-config')
> > -    compile libs.retrofit
> > +    compile libs.retrofit.retrofit
> > +    compile libs.retrofit.gson
>
> we really should convert this to the nebula dependency thing
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/143/files/1aa56f4a6e4e922248ca5fbfc5b5036703bcbb4a#r68301733>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGOdwe48HTFNtaj0GmZwaS2de_ECtJaGks5qOuITgaJpZM4I9Ih5>
> .
>